dev_install.adoc

```diff
@@ -54,13 +54,23 @@ make appstudio-dev-deploy-latest
 If you want to install a local version of any of the Sandbox operators in dev mode then:
 
 . link:quay.adoc[Configure your quay account for dev deployment]
-. Run the following: +
-```
-make dev-deploy-e2e
```

Comment:
should we document also this option ? 
```
# To deploy latest versions of all repositories:
make dev-deploy-e2e
```

Reply:
it's technically the same as `dev-deploy-latest` if no `*_REPO_PATH` is provided. 
`dev-deploy-latest` just sets the requirement for the latest versions explicitly.

Reply:
Right , but I don't see that command documented here either. 

Anyway it's just minor thing!

Reply:
there are many targets in our makefiles that are not documented here. Let's just consider `dev-deploy-e2e` as the middle-target that is called by all other targets that are documented here. 
With additional knowledge, you can use it on its own, but it's not recommended for those who are not familiar with the targets at all.

Reply:
or you mean the `dev-deploy-latest` ? it's documented in the previous section: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/dev_install.adoc#latest-greatest-dev-sandbox

Reply:
Cool ! Ok so we do have documentation about how to deploy the latest versions , in case I don't want/need to deploy my local version. Thanks !